bootstrap/file/file.go

```diff
@@ -0,0 +1,65 @@
+package file
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"os"
+	"time"
+
+	"github.com/edgexfoundry/go-mod-bootstrap/v3/bootstrap/interfaces"
+)
+
+func LoadFile(path string, timeout time.Duration, provider interfaces.SecretProvider) ([]byte, error) {
+	var fileBytes []byte
+	var err error
+
+	client := &http.Client{
+		Timeout: timeout,
+	}
+
+	parsedUrl, err := url.Parse(path)
+	if parsedUrl.Scheme == "http" || parsedUrl.Scheme == "https" {
+		// resp, err := http.Get(path)
+		req, err := http.NewRequest("GET", path, nil)
+
+		// Get httpheader secret
+		params := parsedUrl.Query()
+		if len(params) > 0 {
+			secrets, err := provider.GetSecret(params.Get("edgexSecretName"))
+			if err != nil {
+				return nil, err
+			}
+
+			// Set request header
+			if len(secrets) > 0 {
```

Comment:
Also must verify the data contains the expected item, including `type` = `httpheader`